tests/test_com_aux.py

```diff
@@ -183,9 +182,15 @@ def test_receive_message_exception(mocker, com_aux_init, caplog):
     )
 
 
-def test_receive_message_none(mocker, com_aux_init):
+@pytest.mark.parametrize("clear_buffer", [True, False])
```

Comment:
You are testing that you empty the current queue_out content. But you are using a test where your queue is empty ??

If you have a specific test case please declare a new test function.

Reply:
changed u test